Multiple hypha commands

matthijskooijman opened the pull request:
This is a fix for #323, to support adding `hyphaCommands` (login/logout/index/settings/new page) in multiple places on the page.

tammoterhark commented:
Logged in to test.destadsbron.nl
Theme change to default.
Theme copy to "testtheme"
Hypha commands positioned at multiple positions on pahe in hypha.html
Tested the buttons
Success
Hurray

laurensmartina commented on index.php:
```diff
@@ -136,16 +136,16 @@
 
 	// add hypha commands and navigation
-	$_cmds[] = '<a href="index/'.$hyphaLanguage.'">'.__('index').'</a>';
+	$_cmds[] = '<a clas="index" href="index/'.$hyphaLanguage.'">'.__('index').'</a>';
```
Typo in clas, should be class

matthijskooijman replied:
Fixed

matthijskooijman commented:
Fixed typo and rebased on top of master, merging next.

This still needs some changes to the Stadsbron CSS to take advantage of this feature, @tammoterhark, can I leave those to you? You had an idea how you wanted to style this. If you need anything else code-wise, let me know.
